science/pooler/Portfile

```diff
@@ -27,6 +27,9 @@ maintainers         {@ssb22} \
                     openmaintainer
 
 build.dir           ${worksrcpath}/pooler
+depends_lib         port:libomp
+depends_build       port:clang-16
+configure.cc        clang-mp-16
```

Comment:
Please see https://trac.macports.org/wiki/CompilerSelection#Parallelism and https://trac.macports.org/wiki/CompilerSelection#GettingtheRightCompilerinthePortfile

`compiler.openmp_version …` should be set instead; that should automatically take care of the compiler and libomp dependencies. Depending on what OpenMP version is needed, you may also need to specify `configure.cppflags-append -fopenmp-version=…`.

Reply:
Thanks, now updated to use `compiler.openmp_version` instead